Sources/Optimizely+Decide/OptimizelyUserContext.swift

```diff
@@ -0,0 +1,136 @@
+/****************************************************************************
+* Copyright 2020, Optimizely, Inc. and contributors                        *
+*                                                                          *
+* Licensed under the Apache License, Version 2.0 (the "License");          *
+* you may not use this file except in compliance with the License.         *
+* You may obtain a copy of the License at                                  *
+*                                                                          *
+*    http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                          *
+* Unless required by applicable law or agreed to in writing, software      *
+* distributed under the License is distributed on an "AS IS" BASIS,        *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+* See the License for the specific language governing permissions and      *
+* limitations under the License.                                           *
+***************************************************************************/
+    
+import Foundation
+
+/// An object for user contexts that the SDK will use to make decisions for.
+public class OptimizelyUserContext {
+    weak var optimizely: OptimizelyClient?
+    var userId: String
```

Comment:
userId and attributes should be public.

Reply:
@yasirfolio3 Sounds like you need to access them directly for FSC? public getter access is enough? Any case you need to directly set?

Reply:
yes, no need for setter access, just require public access for FSC to verify the values after each `decide` call 👍

Reply:
@yasirfolio3 getters added for userId + attributes